Makefile

```diff
@@ -16,7 +16,7 @@ CC1       := tools/agbcc/bin/agbcc$(EXE)
 CPP       := $(PREFIX)cpp
 LD        := $(PREFIX)ld
 OBJCOPY   := $(PREFIX)objcopy
-SHA1SUM   := sha1sum -c
+SHA1SUM   := $(shell { command -v sha1sum || command -v shasum; } 2>/dev/null) -c
```

Comment:
Not necessary if you install `md5sha1sum` from Homebrew

Reply:
this too